osbs/build/build_request.py

```diff
@@ -99,6 +99,11 @@ def set_params(self, **kwargs):
         :param auto_build_node_selector: dict, a nodeselector for auto builds
         :param isolated_build_node_selector: dict, a nodeselector for isolated builds
         :param is_auto: bool, indicates if build is auto build
+        :param parent_images_digests: dict, mapping image names with tags to platform specific
```

Comment:
maybe we should just delete it from build_request and just keep it (with that example) in build_requestv2,
since only brv2 will use that param anyway

Reply:
It is deprecated, but still supported. IMO we should keep it. I don't think that removal of it from v1 will help in future consolidation.